backend/uclapi/oauth/models.py

```diff
@@ -0,0 +1,61 @@
+from django.db import models
+from .app_helpers import generate_user_token
+
+class OAuthScope(models.Model):
+    # We should really have a primary key for the OneToOneField
+    id = models.AutoField(primary_key=True)
```

Comment:
It's just a consistency thing I guess. Also remember the bulk_create problem we had a while back? that was resolved by making this field implicit.